### DISCUSSION TITLE
Cannot Update Feed or Mark as Read

### DISCUSSION THREAD

**Discussion by jsphnecclia** [FreshRSS/FreshRSS#5858] [Q&A]
I managed to break a working installation somehow.

Lots of broken functionality -- Cannot Update Feed or Feeds or Mark anything as read.

Everything is green on the installation check page.

When I click to refresh feeds it says 'There are no feeds to refresh'. When I click Mark all as Read, It says "The selected articles have been marked as read", but does nothing.

My first thought is that it is a permissions error, but I checked that (and the installation check page permissions section is all green)

**Comment by Alkarex** [comment]
What database are you using?
Check the write access rights in the data folder and subfolders, in particular `data/users/you/*`

**Reply by jsphnecclia** [reply]
postgres

The permissions are right -- I even changed them to a=rwx temporarily to test if it was a permissions issue

The contents of data/users/me is only 3 files... config.php, config.php.bak.php, and log.txt. Is that right?

**Reply by Alkarex** [reply]
Yes, those are the expected files (except if you use SQLite, in which case there is an additional .db file).
Try to check your PostgreSQL logs, and your FreshRSS logs

**Comment by jsphnecclia** [comment]
Issue was in System Time mismatch. All good now!

**Comment by jsphnecclia** [comment]
Nevermind on that, the issue persists.

Now when I click to refresh feeds it says 'There are no feeds to refresh'. However, when I click Mark all as Read, It says "The selected articles have been marked as read", and it actually marks the items as read..

**Reply by Alkarex** [reply]
Feeds cannot be refreshed too often. Try again, now that a couple of hours have passed.

**Reply by jsphnecclia** [reply]
Thank you. I'm still getting the 'no feeds to refresh' error

If I try to refresh by feed by clicking the gear icon next to the feed name, it says "<feed name> has been updated", but no new entries are shown

**Reply by Alkarex** [reply]
There are probably still some write access issues. Check your Web server logs, FreshRSS logs, and database logs.
But if you can, give a try to our ready-to-use Docker image, so that you do not have to manually deal with too many things https://hub.docker.com/r/freshrss/freshrss/
